pkg/sdk/masking_policy.go

```diff
@@ -346,6 +373,7 @@ func (row maskingPolicyDetailsRow) toMaskingPolicyDetails() *MaskingPolicyDetail
 		ReturnType: dataType,
 		Body:       row.Body,
 	}
+	// TODO (after merged changes in row access policies) use/merge with parsing in row access policies
```

Comment:
Leaving a comment as a reminder (will resolve after the upcoming change)

Reply:
Ok, done. I adjusted this also for row access policies.